website/src/_posts/2021-08-2.0.md

```diff
@@ -127,7 +127,15 @@ Other notable upgrades include browserify to `v10`, typescript to `v4.3`, autopr
 
 [`@uppy/core`][core] provided a `Plugin` class for creating plugins. This was used for any official plugin, but also for users who want to create their own custom plugin. However, `Plugin` always came bundled with Preact, even if the plugin itself didn't add any UI elements.
 
-As of Uppy 2.0.0, `Plugin` has been replaced with `BasePlugin` and `UIPlugin`. `BasePlugin` is the minimum you need to create a plugin and `UIPlugin` adds Preact for rendering user interfaces.
+As of Uppy 2.0.0, `Plugin` has been replaced with `BasePlugin` and `UIPlugin`. `BasePlugin` is the minimum you need to create a plugin and `UIPlugin` adds Preact for rendering user interfaces. If you are not using `UIPlugin` and don't want to bundle its dependencies (such as Preact) with your application, you can cherry-pick your imports to make sure `UIPlugin` doesn't end up in your bundle:
+<!-- eslint-skip -->
+```js
+// UIPlugin may be bundled as it's exported by `@uppy/core`.
+import { Uppy, BasePlugin } from '@uppy/core'
+// Instead, you can import the classes individually.
+import Uppy from '@uppy/core/lib/Uppy.js'
+import BasePlugin from '@uppy/core/lib/BasePlugin.js'
+```
```

Comment:
```suggestion
As of Uppy 2.0.0, `Plugin` has been replaced with `BasePlugin` and `UIPlugin`. `BasePlugin` is the minimum you need to create a plugin and `UIPlugin` adds Preact for rendering user interfaces. 

**Note:** some bundlers will include `UIPlugin` (and therefor Preact) if you import from `@uppy/core`. To make sure this does not happen, you can import `Uppy` and `BasePlugin` directly: 

<!-- eslint-skip -->
```js
import Uppy from '@uppy/core/lib/Uppy.js'
import BasePlugin from '@uppy/core/lib/BasePlugin.js'
```
```

Reply:
I think it should `therefore` here: https://www.grammarly.com/blog/therefore-vs-therefor/